### ISSUE TITLE
process about clicks more than 4 points

### ISSUE THREAD

**Issue by haoyuying** [scaelles/DEXTR-PyTorch#34]
Could you release the training code when clicks more than 4 points, thank you so much.

**Comment by kmaninis** [comment]
You could change the [extreme_points](https://github.com/scaelles/DEXTR-PyTorch/blob/master/dataloaders/custom_transforms.py#L160) function which is gives you the [extreme points of a mask](https://github.com/scaelles/DEXTR-PyTorch/blob/352ccc76067156ebcf7267b07e0a5e43d32e83d5/dataloaders/helpers.py#L138) with a new function that returns more than 4 points.
